### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by NicoOrdax**
Changed "consise" to "concise" in line 382

### CODE REVIEW

**Comment by davewhipp** [comment]
Thanks for catching this @NicoOrdax. I'll merge this in tomorrow after the lesson 4 materials have been merged. Regarding acknowledgements, would you like us to thank you with your name or GitHub username?

**Comment by NicoOrdax** [comment]
> Thanks for catching this @NicoOrdax. I'll merge this in tomorrow after the lesson 4 materials have been merged. Regarding acknowledgements, would you like us to thank you with your name or GitHub username?

Sure thanks, my name is Nicolás Ordax Sommer, or my GitHub username is NicoOrdax :)

**Review by davewhipp — APPROVED** [review]
Thanks again for reviewing the text!

**Pull request merged**
